securedrop/source_templates/lookup.html

```diff
@@ -53,65 +42,61 @@ <h2 class="headline">{{ gettext('Submit Messages') }}</h2>
     </div>
   </div>
 
-  <hr class="no-line">
-  <div class="pull-right">
-    <button type="submit" id="submit-doc-button">
+  <div class="pull-right section-spacing">
+    <button type="submit" id="submit-doc-button" aria-label="{{ gettext('Submit') }}">
       {{ gettext('SUBMIT') }}
     </button>
 
 
 
-    <a href="{{ url_for('main.lookup') }}" class="btn secondary" id="cancel">
-      <img class="icon off-hover" src="{{ url_for('static', filename='i/x_icon-button_blue.png') }}" alt="" width="9" height="11">
-      <img class="icon on-hover" src="{{ url_for('static', filename='i/x_icon-grimace_blue.png') }}" alt="" width="9" height="11">
+    <a href="{{ url_for('main.lookup') }}" class="btn secondary" id="cancel" role="button" aria-label="{{ gettext('Cancel') }}">
       {{ gettext('CANCEL') }}
     </a>
   </div>
 </form>
+</section>
 
-<hr class="no-line">
-
-<h2 class="headline">{{ gettext('Read Replies') }}</h2>
+<section aria-labelledby="replies-heading">
+<h2 id="replies-heading" class="headline">{{ gettext('Read Replies') }}</h2>
 
 <div id="replies">
   {% if replies %}
-    <aside>
-      <p>{{ gettext("You have received a reply. To protect your identity in the unlikely event someone learns your codename, please delete all replies when you're done with them. This also lets us know that you are aware of our reply. You can respond by submitting new files and messages above.") }}</p>
-    </aside>
-    <hr class="no-line">
+    <output role="status">
+      {{ gettext("You have received a reply. To protect your identity in the unlikely event someone learns your codename, please delete all replies when you're done with them. This also lets us know that you are aware of our reply. You can respond by submitting new files and messages above.") }}
+    </output>
     {% for reply in replies %}
-      <div class="reply">
-        <time class="date" title="{{ reply.date|rel_datetime_format }}" datetime="{{ reply.date }}">{{ reply.date|rel_datetime_format(relative=True) }}</time>
+      <article class="reply" aria-labelledby="timestamp-{{ reply.filename }}">
+        <time id="timestamp-{{ reply.filename }}" class="date" title="{{ reply.date|rel_datetime_format }}" datetime="{{ reply.date }}" aria-hidden="true">{{ reply.date|rel_datetime_format(relative=True) }}</time>
```

Comment:
This is probably another Orca issue, but I can't get to the timestamps.

Reply:
Nope; this was me being overzealous in my ARIA annotations because macOS VoiceOver was being overzealous in its narration.  ;-)  Simplified and streamlined, in both markup and narration, in 08cce2cf6f85c8319c57c07dea3373b8f1ccff1a.

Reply:
After wrapping these in `h3`s, Orca is reading the timestamps.